app/scripts/templates/sign_in.mustache

```diff
@@ -19,6 +19,9 @@
     {{^error}}
       <div class="error"></div>
       <div class="success"></div>
+        {{#migration}}
+          <div class="info nudge">{{#t}}Already have a Firefox Account? Migrate your sync data by signing into it here.{{/t}}</div>
```

Comment:
@ryanfeeley - I know you always have good reasons, how come there is markedly different text between the signin and signup pages? Can the two pages have similar roots that diverge w/ the action taken?

signup: `Migrate your Sync data by creating a new Firefox Account`
signin: `Migrate your Sync data by signing in to an existing Firefox Account`